src/DependencyInjection/Configuration.php

```diff
@@ -127,6 +127,11 @@ public function getConfigTreeBuilder()
                                         ->defaultValue(AmqpConnectionFactory::class)
                                         ->info('This option defines an AMQP connection factory to be used to establish a connection with RabbitMQ.')
                                     ->end()
+                                    ->scalarNode('delay_strategy_class')
+                                        ->cannotBeEmpty()
+                                        ->defaultValue(null)
+                                        ->info('This option defines the Delay Strategy to use in case an AMQP connection factory is of DelayStrategyAware type. Defaults to null if should be ignored.')
```

Comment:
```suggestion
                                        ->info('This option defines the delay strategy to use in case an AMQP connection factory is of DelayStrategyAware type. Defaults to null if should be ignored.')
```